mlflow/server/js/src/experiment-tracking/components/MetricsPlotPanel.js

```diff
@@ -523,7 +523,10 @@ const mapStateToProps = (state, ownProps) => {
   // Flat array of all metrics, with history and information of the run it belongs to
   // This is used for underlying MetricsPlotView & predicting chartType for MetricsPlotControls
   const metricsWithRunInfoAndHistory = _.flatMap(runUuids, (runUuid) => {
-    const runDisplayName = Utils.getRunDisplayName(getRunTags(runUuid, state), runUuid);
+    const runDisplayName = Utils.getRunDisplayName(
+      getRunTags(runUuid, state),
+      runUuid.slice(0, 6) + '...'
```

Comment:
We were using `Utils.truncateString(runDisplayName, MAX_RUN_NAME_DISPLAY_LENGTH)`, could you please check if you can achieve this by use and decrease MAX_RUN_NAME_DISPLAY_LENGTH?